server/src/test/api/profile/handler/get-api-credentials-handler.test.ts

```diff
@@ -0,0 +1,103 @@
+import { StatusCodes } from 'http-status-codes'
+import { Request, Response } from 'express'
+import { getApiCredentialsHandler } from '../../../../api/profile/handler/get-api-credentials-handler'
+import { getClientCredentialsQueryHandler } from '../../../../application/profile/query-handler/getClientCredentialsQueryHandler'
+
+// Mock the dependencies
+jest.mock('../../../../application/profile/query-handler/getClientCredentialsQueryHandler')
+
+const mockGetClientCredentialsQueryHandler = getClientCredentialsQueryHandler as jest.MockedFunction<
+  typeof getClientCredentialsQueryHandler
+>
```

Comment:
There's a handy method that jest provides:
```suggestion
const mockGetClientCredentialsQueryHandler = jest.mocked(getClientCredentialsQueryHandler)
```

Reply:
Thanks. Updated.